rasa/core/channels/slack.py

```diff
@@ -211,7 +237,7 @@ def _is_user_message(slack_event: Dict) -> bool:
         )
 
     @staticmethod
-    def _sanitize_user_message(text, uids_to_remove) -> Text:
+    def _sanitize_user_message(text, uids_to_remove: Optional[List[Text]]) -> Text:
```

Comment:
thanks for all the typing!
```suggestion
    def _sanitize_user_message(text: Text, uids_to_remove: Optional[List[Text]]) -> Text:
```